server/routes/api/users/schema.ts

```diff
@@ -84,6 +84,7 @@ export const UsersUpdateSchema = BaseSchema.extend({
     avatarUrl: z.string().nullish(),
     language: zodEnumFromObjectKeys(locales).optional(),
     preferences: z.record(z.nativeEnum(UserPreference), z.boolean()).optional(),
+    timezone: z.string().optional(),
```

Comment:
Shall we validate the value? There are custom zod validators in `server/utils/zod` 

https://stackoverflow.com/a/44118363